content/doc/pipeline/index.adoc

```diff
@@ -314,7 +316,7 @@ SCM system, and then loading those scripts into Jenkins using the *Pipeline
 Script from SCM* option enabled by the workflow-scm-step plugin, which is one of
 the plugins that the Pipeline plugin depends on and automatically installs.
 
-Loading pipeline scripts from another source via `checkout scm` leverages the
+Loading pipeline scripts from another source using the `checkout scm` command leverages the
```

Comment:
Thanks, Tyler. I fixed the file and added a little more to it. Hopefully
it's good to go now.

On Wed, May 25, 2016 at 5:42 PM, R. Tyler Croy <notifications@github.com>
wrote:

> In content/doc/pipeline/index.adoc
> <https://github.com/jenkins-infra/jenkins.io/pull/264#discussion_r64658575>
> :
>
> > @@ -314,7 +316,7 @@ SCM system, and then loading those scripts into Jenkins using the *Pipeline
> >  Script from SCM* option enabled by the workflow-scm-step plugin, which is one of
> >  the plugins that the Pipeline plugin depends on and automatically installs.
> >
> > -Loading pipeline scripts from another source via `checkout scm` leverages the
> > +Loading pipeline scripts from another source using the `checkout scm` command leverages the
>
> Generally the documentation uses the noun step instead of command when
> referring the Pipeline steps.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/264/files/201720a4cc47b1f82e413dbdc183bdebe867cf27#r64658575>
>